src/index.css

```
@@ -201,14 +235,12 @@
-      rgb(var(--accent-500) / 0) 0%,
-      rgb(var(--accent-500) / 0.95) 20%,
-      rgb(var(--accent-500) / 1) 45%,
-      rgb(var(--accent-500) / 0.95) 70%,
-      rgb(var(--accent-500) / 0) 100%
-    );
+    background: linear-gradient(90deg,
+        rgb(var(--accent-500) / 0) 0%,
+        rgb(var(--accent-500) / 0.95) 20%,
+        rgb(var(--accent-500) / 1) 45%,
+        rgb(var(--accent-500) / 0.95) 70%,
+        rgb(var(--accent-500) / 0) 100%);
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)